README.md

```diff
@@ -7,27 +7,27 @@ The image segmentation workflow is managed using snakemake, a user-friendly pyth
 
 The segmentation workflow consists of the following steps each defined by a "rule". 
 The output of each rule is store to specific folder that is produced by the worrkflow:
-   1. Download the fish **Images** from [Tulane server](http://www.tubri.org/HDR/INHS/) using a simple bash script.
+   1. Download the fish **Images** from [Fish-AIR](https://fishair.org/) using a simple bash script.
       * Images are saved to the folder **Images**.
```

Comment:
Your changes are fine, but the text is misleading. The step just downloads images based on URLs in the CSV file. They could come from anywhere. The CSV needs to have a `path` that is actually a URL and a `original_file_name` that will have the filename extension removed and that will be the unique identifier for the image. We also don't use a bash script.
```suggestion
   1. Download the fish **Images** from URLs within the input CSV file. The CSV file must have a column named `path` that contains the URL of the image. The CSV file must have a column named `original_file_name` that will be used to create a unique filename for the images downloaded.
      * Images are saved to the folder **Images**.
```